### ISSUE TITLE
tokenizer error

### ISSUE THREAD

**Issue by Steel-skull** [cg123/mergekit#133]
this could be due to my own mistake but i cant seem for the life of me figure out why the merge works then the tokenizer merge fails.

```
Traceback (most recent call last):
  File "/usr/local/bin/mergekit-moe", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/content/mergekit/mergekit/options.py", line 76, in wrapper
    f(*args, **kwargs)
  File "/content/mergekit/mergekit/scripts/mixtral_moe.py", line 450, in main
    build(
  File "/content/mergekit/mergekit/scripts/mixtral_moe.py", line 360, in build
    tokenizer = transformers.AutoTokenizer.from_pretrained(base_model.path)
  File "/usr/local/lib/python3.10/dist-packages/pydantic/main.py", line 761, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'ModelReference' object has no attribute 'path'
```

ive attempted with multiple different configs.

**Comment by cg123** [comment]
This should be fixed by c339660a26c6ffb05e9608583cbdc74b799c8922.

**Comment by Steel-skull** [comment]
> This should be fixed by [c339660](https://github.com/cg123/mergekit/commit/c339660a26c6ffb05e9608583cbdc74b799c8922).

it is thank you
